### ISSUE TITLE
confirmation handler

### ISSUE THREAD

**Issue by jeesunhub** [SmartThingsCommunity/smartapp-sdk-nodejs#176]
There is no confirmation 'handler' in smartapp.js
below is the code in confirmation part.

```
  case 'CONFIRMATION': {
    if (evt.confirmationData && evt.confirmationData.appId && evt.confirmationData.confirmationUrl) {
	if (this._id) {
	  if (evt.confirmationData.appId === this._id) {
	    this._log.info(`CONFIRMATION request for app ${evt.confirmationData.appId}, to enable events visit ${evt.confirmationData.confirmationUrl}`)
	  } else {
	    this._log.warn(`Unexpected CONFIRMATION request for app ${evt.confirmationData.appId}, received ${JSON.stringify(evt)}`)
	  }
	} else {
	  this._log.info(`CONFIRMATION request for app ${evt.confirmationData.appId}, to enable events visit ${evt.confirmationData.confirmationUrl}`)
        }
    } else {
      this._log.warn(`Invalid CONFIRMATION request ${JSON.stringify(evt)}`)
    }
  break
}
```

others have handlers.
so, I wonder if there is a plan to support confirmation handler.

**Comment by bflorian** [comment]
Yes, we will look into adding a handler for this event.

The original thought was that the default handling of this event would be sufficient, since information it typically returns such as initial page and permissions can be set with their own individual options, but we have encountered cases such as the desire to dynamically select the initial page or permissions which would require a handler. 

If you have any specific use-cases for what you would want to be able to do in this handler, feel free to add them to this issue.

**Comment by bflorian** [comment]
Fixed in version 2.5.0
